tests/integration/pilot/common/traffic.go

```diff
@@ -19,93 +19,82 @@ import (
 	"fmt"
 	"time"
 
+	"istio.io/istio/pkg/test"
 	echoclient "istio.io/istio/pkg/test/echo/client"
 	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo"
 	"istio.io/istio/pkg/test/util/retry"
 )
 
 // callsPerCluster is used to ensure cross-cluster load balancing has a chance to work
 const callsPerCluster = 5
 
+var (
+	// Custom retry options for traffic calls.
+	retryOptions = []retry.Option{retry.Delay(1000 * time.Millisecond), retry.Converge(3)}
```

Comment:
why such a larger delay? And why not align with the global defaults which can be configured by flag?

Reply:
@stevenctl do you know why the traffic tests were using a 1s delay?

Reply:
Looked through the commit history and added a comment.